### DISCUSSION TITLE
Does this work auto-switching to an internal portal?

### DISCUSSION THREAD

**Discussion by adyfoot** [yuezk/GlobalProtect-openconnect#383] [General]
Hi there - really excited that I might finally be able to switch to Linux and have a working GlobalProtect solution for User-ID.  I have configured GlobalProtect to auto detect an internal network and switch accordingly - this works great on the official GP client.  Does this work on your OpenConnect version too please?

**Comment by yuezk** [comment]
Supported in 2.3.4 and worked per the feedback in https://github.com/yuezk/GlobalProtect-openconnect/issues/377. You can upgrade it and give it a try.
